Tests/Brokerages/Samco/SamcoBrokerageTests.cs

```diff
@@ -0,0 +1,690 @@
+﻿using Moq;
+using NUnit.Framework;
+using QuantConnect.Brokerages;
+using QuantConnect.Brokerages.Samco;
+using QuantConnect.Configuration;
+using QuantConnect.Data;
+using QuantConnect.Data.Market;
+using QuantConnect.Interfaces;
+using QuantConnect.Lean.Engine.DataFeeds;
+using QuantConnect.Logging;
+using QuantConnect.Orders;
+using QuantConnect.Securities;
+using QuantConnect.Tests.Common.Securities;
+using System;
+using System.Diagnostics;
+using System.Globalization;
+using System.Linq;
+using System.Threading;
+
+namespace QuantConnect.Tests.Brokerages.Samco
+{
+    [TestFixture, Ignore("This test requires a configured and active Samco account")]
+    public class SamcoBrokerageTests
```

Comment:
We have a base brokerage test setup at `BrokerageTests` Samco should inherit from it 👍, it can have additional tests too. Can follow other brokerages here 💯